client/src/components/AdminCard/index.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+import PropTypes from 'prop-types';
+import { Button } from 'antd';
+import './style.css';
+
+class AdminCard extends Component {
+  state = {
+    students: [],
+  };
+
+  async componentDidMount() {
+    const { studentNames } = this.props;
+    if (studentNames) {
+      const result = await axios.get(
```

Comment:
In some cases, you may have an error when fetching data from our endpoint, so you need to handle it using ```try-catch ``` by that the execution is transferred to the catch block and we can handle whatever error is thrown.